core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -216,7 +216,7 @@ abstract class DataContainer extends Backend
 
 	/**
 	 * Active record
-	 * @var \stdClass|null
+	 * @var Model|FilesModel|object|null
```

Comment:
Just a question: why `Model` and `FilesModel` if every `FilesModel` is a `Model?

Reply:
Fixed in 418f447fa067631cc2ba98d09c4d227d45debb05.